Main.py

```diff
@@ -62,14 +62,15 @@
 
 @bot.event
 async def on_ready():
-    if bot.config.log_webhook_id and bot.config.log_webhook_token:
+    if bot.config.dev_log_webhook_id and bot.config.dev_log_webhook_token:
         webhook_string = " and to the log webhook"
     else:
         webhook_string = ""
     sys.stdout.write(f'Bot is ready, program output will be written to a '
                      f'log file{webhook_string}.\n')
     sys.stdout.flush()
-    bot.logger.info('Logged in as {} ({})'.format(bot.user.name, bot.user.id))
+    bot.dev_logger.info('Logged in as {} ({})'.format(bot.user.name,
```

Comment:
we can use f-strings now, thanks yapf!!!!! :)))))))))))

Reply:
Resolved in 380013c